src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequest.java

```diff
@@ -10,9 +17,28 @@
     @JsonProperty("company_number")
     private final String companyNumber;
 
-    public ChipsRequest(String objectionId, String companyNumber) {
+    @JsonProperty("attachments")
+    private final Map<String, String> attachments;
+
+    @JsonProperty("reference_number")
+    private final String referenceNumber;
+
+    @JsonProperty("customer_email")
+    private final String customerEmail;
+
+    @JsonProperty("reason")
+    private final String reason;
+
+    public ChipsRequest(String objectionId, String companyNumber,
+                        List<Attachment> attachments, String referenceNumber,
+                        String customerEmail, String reason,
+                        String dowloadPrefix) {
```

Comment:
dowload typo

Reply:
Never heard of that so put "download" instead.